protocol/v2/qbft/roundtimer/timer.go

```diff
@@ -6,22 +6,48 @@ import (
 	"sync/atomic"
 	"time"
 
+	"github.com/attestantio/go-eth2-client/spec/phase0"
 	specqbft "github.com/bloxapp/ssv-spec/qbft"
+	spectypes "github.com/bloxapp/ssv-spec/types"
 )
 
-type RoundTimeoutFunc func(specqbft.Round) time.Duration
+type TimeAtSlotFunc func(slot phase0.Slot) int64
```

Comment:
its a bit confusing that we're using slot and height here. lets stick to either height or slot.